fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -313,6 +319,96 @@ public void ReportMessage(object sender, MessageEventArgs e)
         {
             MessageReport?.Invoke(sender, e);
         }
+
+        internal List<string> ProduceImpactAreaStructureDetails()
+        {
+            //this list will be the size of the number of structures + 1 where the first string is the header
+            List<string> structureDetails = _inventory.StructureDetails();
+            DeterministicInventory deterministicInventory = _inventory.Sample(new compute.MedianRandomProvider());
+            StagesToStrings(ref structureDetails);
+            DepthsToStrings(deterministicInventory, ref structureDetails);
+            DamagesToStrings(deterministicInventory, STRUCTURE_DAMAGE, ref structureDetails);
+            DamagesToStrings(deterministicInventory, CONTENT_DAMAGE, ref structureDetails);
+            DamagesToStrings(deterministicInventory, OTHER_DAMAGE, ref structureDetails);
+            DamagesToStrings(deterministicInventory, VEHICLE_DAMAGE, ref structureDetails);
+
+            return structureDetails;
+        }
+
+        private void DamagesToStrings(DeterministicInventory deterministicInventory, string assetType, ref List<string> structureDetails)
```

Comment:
I'm a little surprised we don't have an enum for asset type. If so, that might be good to use, or good to generate unless we're purposely leaving this as a string. 

The string of If's could probably be cleaned up into a switch statement, but I think that's just sytlistic.

Reply:
We ended up with properties on asset type. This might be a nice refactor; we'd want to think about how to make use of the same enums in the model as in the view and view model.